docker/docker-compose.yaml

```
@@ -9,6 +9,8 @@ services:
       MODE: api
       # The log level for the application. Supported values are `DEBUG`, `INFO`, `WARNING`, `ERROR`, `CRITICAL`
       LOG_LEVEL: INFO
+      # DEBUG Environment Variable, uncomment to enable DEBUG model and get more logs
```

```suggestion
      # enable DEBUG mode and output more logs
```
1. skip repeating `DEBUG` name
2. fix typo of `model` to `mode`